app/retail/templates/leaderboard.html

```diff
@@ -17,117 +17,133 @@
                   <div class="container winner-content">
                     <div class="row align-items-center">
                       {% if forloop.counter == 1 %}
-                        <div class="col-4 winner-position winner-position-first"></div>
-                      {% else %}
-                      {% if forloop.counter == 2 %}
-                        <div class="col-4 winner-position winner-position-second"></div>
-                      {% else %}
-                      {% if forloop.counter == 3 %}
-                        <div class="col-4 winner-position winner-position-third"></div>
-                      {% endif %}{% endif %}{% endif %}
-
-                      <div class="col-8 " >
-                        <h5 style="color:white">
-                          {% if not is_linked_to_profile %}
-                          <a href="{{podium_item.url}}" target="_blank">
-                            {{podium_item.at_ify_username}}
-                          </a>
-                          {% else %}
-                            {{podium_item.at_ify_username}}
-                          {% endif %}
-                        </h5>
-                        <h2 style="color:#86eec4" > ${{podium_item.amount|floatformat:0}} </h2>
-                      </div>
+                        <!--<div class="col-12 winner-position winner-position-first"></div>-->
+                      {% endif %}
                     </div>
+
                   </div>
+
+                  <h6 class="pt-3 pb-3 text-center restext" style="color:#3f78cd; white-space: nowrap;">
+                    {% if forloop.counter == 1 %}
+                      <img src="/static/v2/images/leaderboard-1.png" style="width: 17.5px; height: 20px; float: left"></img>
+                    {% endif %}
+                    {% if not is_linked_to_profile %}
+                    <a href="{{podium_item.url}}" target="_blank">
+                      {{podium_item.at_ify_username}}
+                    </a>
+                    {% else %}
+                      {{podium_item.at_ify_username}}
+                    {% endif %}
+                  </h6>
+
                 </div>
               {% endfor %}
-            </div>
-            <div class="row mt-0 ml-1 pt-1 pl-1" style="height: 4em; padding: 0.5em;">
-              <div class="col-7" style="padding-top: 0.5em;">
-                <h3>{{title}} <span id="alpha">Alpha</span></h3>
-              </div>
-              <div class="col-2 filter">
-                <div class="form-group">
-                  {% if technologies|length  > 0%}
-                    <select id="tech-keyword" class="form-control tech-options key">
-                      <option value="all">All</option>
-                    </select>
-                  {%endif%}
+          </div>
+
+          <!-- END PODIUM -->
+
+              <div class="offset-md-1 col-sm-12 col-md-10">
+
+                <div class="card shadow-lg">
+                  <div class="card-body">
```

Comment:
Indentation of child div needs to be fixed

Reply:
fixed